samples/multiplatform/androidApp/app/build.gradle

```diff
@@ -21,8 +21,8 @@ android {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'org.greeting:android:1.0'
+    implementation project(':greeting:android')
```

Comment:
I wanted this sample to reflect 100% of the documentation.
If we use this the documentation and the sample is not the same anymore.
Either change the docu or revert this...

## Update
Just seen that there is a big note in the documentation 🙃 
Maybe we don't "note" it anymore but used it as a general rule?!